### PR TITLE
[CWS] Add 7.52 release notes

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Add CWS release note.

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

**Review by YoannGh — APPROVED** [review]

**Review by spikat — APPROVED** [review]

File: releasenotes/notes/csm-7-52-22c6092c5437806b.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    CWS: Add ``chdir`` event allowing recent container escape detection.
```
Comment (maycmlee) [inline releasenotes/notes/csm-7-52-22c6092c5437806b.yaml:4]:
```suggestion
    CWS: Add ``chdir`` event to allow recent container escape detection.
```

File: releasenotes/notes/csm-7-52-22c6092c5437806b.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  - |
+    CWS: [BETA] Add File Integrity Monitoring support on Windows, supporting both files and registry.
+  - |
+    CWS: The Agent will now automatically suppress benign security events if they have already been reported for a particular container image.
```
Comment (maycmlee) [inline releasenotes/notes/csm-7-52-22c6092c5437806b.yaml:8]:
```suggestion
    CWS: The Agent now automatically suppresses benign security events if they have already been reported for a particular container image.
```

**Review by maycmlee — APPROVED** [review]
Left a couple of small suggestions, but approving.

**Pull request merged**